packages/EasyCodingStandard/config/common/spaces.yml

```diff
@@ -59,3 +59,10 @@ services:
     PhpCsFixer\Fixer\FunctionNotation\MethodArgumentSpaceFixer: ~
 
     PHP_CodeSniffer\Standards\Squiz\Sniffs\WhiteSpace\LanguageConstructSpacingSniff: ~
+
+    SlevomatCodingStandard\Sniffs\Classes\TraitUseDeclarationSniff:
```

Comment:
I'm not sure the right places is here is `spaces.yml`. It doesn't work with spaces.

What about `common/control-structures.yml`?

Reply:
Feel free to merge this after this move. 

It's ok for me :+1: